GenericNode/src/main/java/genericnode/Clients/TCPClient.java

```diff
@@ -0,0 +1,71 @@
+package genericnode.Clients;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+import java.util.AbstractMap.SimpleEntry;
+
+public class TCPClient extends Client {
+
+    Socket clientSocket;
+    DataOutputStream outToServer = null;
+    DataInputStream inFromServer = null;
+    @Override
+    public void put(String key, String value) throws IOException {
+        outToServer.writeUTF(key);
+        outToServer.writeUTF(value);
+        clientSocket.close();
+    }
+
+    @Override
+    public String get(String key) throws IOException {
+        outToServer.writeUTF(key);
+        System.out.println(inFromServer.readUTF());
```

Comment:
Format of the output should be modified. 
I think the client add "server response:" to the response from the server, while the server respond in a format like "get key=a val=123"
$ java -jar GenericNode.jar tc localhost 1234 get a
server response:get key=a get val=123

Reply:
Great I didnt know there was a specific format. I will do the changes everywhere required